tests/vue/text-editor/tokenize-options-block.test.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-env jest */
+/* eslint-disable no-unused-vars */
+
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import AppController from '@/lib/controllers/app-controller.js'
+import TokenizeOptionsBlock from '@/vue/text-editor/tokenize-options-block.vue'
+import OptionItemBlock from '@/vue/options/option-item-block.vue'
+
+import Vuex from "vuex"
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+let appC, localTextEditorOptions
+
+describe('tokenize-options-block.test.js', () => {
+  console.error = function () {}
+  console.log = function () {}
+  console.warn = function () {}
+
+  beforeEach(async () => {
+    jest.spyOn(console, 'error')
+    jest.spyOn(console, 'log')
+    jest.spyOn(console, 'warn')
+
+    appC = new AppController({
+      appId:'alpheios-alignment-editor'
+    })
+    
+    appC.defineStore()
+    appC.defineL10Support()
+    appC.defineNotificationSupport(appC.store)
+
+    await appC.defineSettingsController()
+    await appC.settingsC.init()
+    await appC.settingsC.uploadRemoteSettings()

```

Comment:
per previous comment, please eliminate dependency upon remote tokenization service for these tests.

Reply:
Fixed